pkgs/applications/editors/lapce/default.nix

```diff
@@ -22,74 +23,41 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "lapce";
-  version = "0.2.8";
+  version = "0.3.0";
 
   src = fetchFromGitHub {
     owner = "lapce";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-cfQQ+PaInUB6B61sZ9iS/zt3L6Vc/vPOJTtEwR0BLco=";
+    sha256 = "sha256-5HJ/ht5L1M6KWsQST2b36kz5RDiG2rMXOMS6O5H9R30=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
```

Comment:
This can be changed to ``lockFile = "${src}/Cargo.lock";`` and then you don't need to include the Cargo.lock in nixpkgs.

Reply:
`lockFile = "${src}/Cargo.lock" will cause [IFD](https://nixos.org/manual/nix/unstable/language/import-from-derivation)